play/src/pages/JoinGame.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import { useTheme } from '@mui/material/styles';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import SplashScreen from '../components/joingame/SplashScreen';
+import { JoinGameState } from '../lib/PlayModels';
+
+interface JoinGameProps {
+  joinGameState: JoinGameState;
+}
+
+export default function JoinGame({ joinGameState }: JoinGameProps) {
+  const theme = useTheme();
+  const isMobileDevice = useMediaQuery(theme.breakpoints.down('sm'));
+  const [inputError, setInputError] = useState(true); // eslint-disable-line @typescript-eslint/no-unused-vars
+  const [gameCodeValue, setGameCodeValue] = useState('####');
+  const [playerFirstName, setPlayerFirstName] = useState('First Name');
+  const [playerLastName, setPlayerLastName] = useState('Last Name');
+  const [avatar, setAvatar] = useState(0); // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
why include the unused vars?

Reply:
Laziness honestly. I know these are coming in subsequent screens. Good call though, they have been removed.